plone-5/instance/src/ukstats.csv_type/src/ukstats/csv_type/csv.py

```diff
@@ -0,0 +1,13 @@
+from ukstats.csv_type import _
+from plone.supermodel import model
+from plone.namedfile.field import NamedBlobFile
+from zope import schema
+
+
+class ICSV(model.Schema):
+    """A conference program. Programs can contain Sessions."""
+    csv = NamedBlobFile(
+        title=_(u'csv'),
```

Comment:
the title looks a bit weird in the UI (esp. the Volto one)?
![Screenshot 2022-03-22 at 17 00 42](https://user-images.githubusercontent.com/910448/159534990-f3f92edb-0e63-430a-8c46-1e01ee3dc0ab.png)
![Screenshot 2022-03-22 at 17 00 57](https://user-images.githubusercontent.com/910448/159534999-3d6b0182-3ad6-43d4-86d1-1883f3de0545.png)